webpack.config.js

```diff
@@ -33,9 +33,7 @@ Encore
      */
     .addEntry('app', './assets/app.js')
     .addStyleEntry('styles', './assets/styles/styles.css')
-    .addStyleEntry('footer', './assets/styles/components/footer.css')
-
-    .addStyleEntry('styles', './assets/styles/styles.css')
+    .addStyleEntry('footer', './assets/styles/components/navbar.css')
```

Comment:
if you want you can include all global and layout css files into app.js and it will automatically be charged thanks to the entry_link_tags option